controller/fileStore.go

```diff
@@ -85,7 +85,11 @@ func (fs *FileStore) fileStoreService() {
 		case WRITE:
 			response.error = ioutil.WriteFile(path.Join(fs.root, req.fileName), req.fileContents, 0600)
 		case DELETE:
-			response.error = os.Remove(path.Join(fs.root, req.fileName))
+			response.error = nil
+			filePath := path.Join(fs.root, req.fileName)
+			if _, err := os.Stat(filePath); os.IsExist(err) {
```

Comment:
@soamvasani thanks for your suggestion! I just updated the commit. By the way, should I squash the commits into one?